examples/mobile/HomeApp/js/app.js

```diff
@@ -75,8 +52,20 @@
 	}
 
 	function onPopupSubmit() {
-		getWidgetsState();
-		toggleWidgetsState();
+
+		activeWebClipList = [];
+		webClipList.forEach(function (webclip) {
+			var webClipName = webclip;
+
+			webClipName = webClipName.replace("webclip/", "");
+			webClipName = webClipName.replace("/", "");
```

Comment:
Isn't it cleaner to write these to expressions as a one?
``` js
webClipName = webClipName.replace("webclip/", "").replace("/", "");
```

Reply:
done